source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -524,6 +523,11 @@ namespace AdaptiveCards
                                                  AdaptiveRenderArgs renderArgs);
             };
```

Comment:
nit, no need for `;` here or on any of the `runtimeclass` or `interface` uses below.